src/main/java/meditracker/argument/ArgumentName.java

```diff
@@ -11,7 +11,8 @@ public enum ArgumentName {
     LIST_INDEX("listIndex"),
     NAME("name"),
     QUANTITY("quantity"),
-    REMARKS("remarks");
+    REMARKS("remarks"),
+    LIST_TYPE("list");
```

Comment:
```suggestion
    LIST_TYPE("listType");
```
The value of the enum should follow the same convention

Reply:
Got it. Right away.